index.html

```diff
@@ -0,0 +1,234 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet">
+<link rel="icon" type="image/png" href="./images/favicon.png">
+
+<title>Capstone-Module1</title>
+</head>
+<body>
+
+    <!-- Mobile menu Popup-->
+
+    <div class="menu-popup" id="mob-menu">
+      <i class="fas fa-times x-icon" id="x-iconid"></i>
+      <div class="menu-popup-links grey-txt" id="navLinks">
+        <a href="index.html">Home</a>
+        <a href="about.html">About</a>
+        <a href="#">Program</a>
+        <a href="#">Join</a>
+        <a href="#">Sponsor</a>
+        <a href="#">News</a>
+      </div>
+    </div>
+
+<!-- Main page Section -->
+
+    <header>
+        <!-- below tags just appears in desktop -->
+
+        <aside class="aside-section">
+            <div class="top-elements">
+                <i class="fa-brands fa-facebook-f aside-icon"></i>
+                <i class="fa-brands fa-twitter aside-icon"></i>
+                <a href="#">English</a>
+                <a href="#">Page</a>
+                <a href="#">Logout</a>
+            </div>
+        </aside>
+        <div class="desktop-menu">
+            <div class="main-Dmenu">
+                <div class="Dmain-logo">
+                    <a href="index.html">
+                        <img src="./images/shaheen.png" alt="">
+                    </a>
+                </div>
+                <nav class="Dmain-nav">
+                    <a href="about.html">About</a>
+                    <a href="#">Program</a>
+                    <a href="#">Join</a>
+                    <a href="#">Sponsor</a>
+                    <a href="#">News</a>
+                    <a href="#" class="Dlive">Airways</a>
+                </nav>
+            </div>
+        </div>
+
+        <!-- above tags just appears in desktop -->
+        <div class="fst-header-container">
+            <div class="fst-Headerimg">
+                <i class="fa-solid fa-bars hamburger" id="hamburger-icon"></i>  
+            </div>
+        </div>
+    </header>
```

Comment:
Hi, @andersonlebon thanks for taking the time and reviewing my project. I am not getting this issue while running the pages, even when i used the deploy link there it's showing too it might be taking time to load as it is doing so at some time.

Reply:
![hamburger](https://user-images.githubusercontent.com/54817285/182011116-7dddb6bf-727c-4c88-a6b8-7b4a64952d48.png)